src/components/header/header.module.css

```diff
@@ -1,4 +1,128 @@
-.header {
-  font-size: 34px;
-  text-align: center;
+.headerContainer {
+  display: flex;
+  align-items: center;
+  min-height: 80px;
+}
+
+.logo {
+  flex-grow: 2;
```

Comment:
Why do you set here 2?  And 20 for `appLayout`?
As a result, the content of the right block doesn't fi to its container
![image](https://github.com/volodymyr-kravchenko-moc/fe-react-2024/assets/98031768/3e5f2d45-51f0-4266-98bc-0878f2972d36)

I suggest removing all these `flex-grow` props, fix all margins for header blocks and make `margin-left: auto` for `navBar`. It eats all free space

Reply:
I was playing around with flexes to understand how they work a bit better, so that's why the header layout turned out a bit strange. Thanks for the advice, I fixed it